components/nexusoperations/config.go

```diff
@@ -149,6 +149,23 @@ var RetryPolicyMaximumInterval = dynamicconfig.NewGlobalDurationSetting(
 	`The maximum backoff interval between every nexus StartOperation or CancelOperation request for a given operation.`,
 )
 
+var MetricTagConfiguration = dynamicconfig.NewNamespaceTypedSetting(
```

Comment:
Make this a global typed setting please so it can be properly cached with `NewGlobalCachedTypedValue` later. This is a limitation of the dynamic config implementation today.